ClientApp/viking-vault/src/AdminDashboard/UserData.tsx

```diff
@@ -1,14 +1,13 @@
 import React from "react";
-import '../components/styles.css';
+import '../Resources/Components/styles.css';
```

Comment:
Why is there a "Components" folder in Resources?

Reply:
For components that are used in other components. I could renamed it to "commons" if it would be more descriptive

Reply:
That would be way better. Resources usually holds exactly that. Resources (i.e. images, json files and other stuff like that). You should have a "Common" folder only with components that are shared.